src/Symfony/Component/Process/Process.php

```diff
@@ -515,17 +512,12 @@ public function isSuccessful()
      *
      * @return bool
      *
-     * @throws RuntimeException In case --enable-sigchild is activated
-     * @throws LogicException   In case the process is not terminated
+     * @throws LogicException In case the process is not terminated
      */
     public function hasBeenSignaled()
     {
         $this->requireProcessIsTerminated(__FUNCTION__);
 
-        if ($this->isSigchildEnabled()) {
-            throw new RuntimeException('This PHP has been compiled with --enable-sigchild. Term signal can not be retrieved.');
```

Comment:
we are able to retrieve it when enhancing compat, but can we also without enhancing it ?